about.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/4f02d055e5.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./styles.css">
+    <title>About Proclaim</title>
+  </head>
+  <body>
+    <section class="about-container">
+      <header class="header">
+        <nav class="navbar ">
+          <ul class="top-nav">
+
+
+            <li class="top-nav-item"><span class="material-icons material-icons-outlined">
+                search
+              </span>
+            </li>
+            <li class="top-nav-item">English
+            </li>
+            <li class="top-nav-item">MyPage
+            </li>
+            <li class="top-nav-item">Logout
+            </li>
+
+          </ul>
+        </nav>
+        <nav class="navbar">
+          <ul class="nav">
+            <li class="nav-menu">
+              <a href="#"><span class="material-icons menu ">
+                  menu
+                </span></a>
+            </li>
+            <li>
+              <a href="index.html"> <img src="./images/logo.webp" alt="logo" class="logo">
+              </a>
+            </li>
+
+            <li class="nav-item">
+              <a href="about.html">About
+              </a>
+            </li>
+            <li>
+              <a href="#" class="nav-item">Program
+              </a>
+            </li>
+            <li>
+              <a href="#" class="nav-item">Join
+              </a>
+            </li>
+            <li>
+              <a href="#" class="nav-item">Sponsor
+              </a>
+
+            </li>
+            <li class="nav-item">
+              <a href="#">News
+              </a>
+            </li>
+            <li class="nav-item campaign">
+              <a href="#"> campaign
+              </a>
+            </li>
+          </ul>
+        </nav>
+
+      </header>
+      <section class="landing">
+        <h2 class="heading heading-about">"Hello Proclaimers"</h2>
+        <h1 class="heading custom"> To the ends of The world. <br> Proclaim conference 2021</h1>
```

Comment:
- [ ] Kindly fix the hierarchy issue. It's important in following best practices that H1 comes before H2.